### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by KeewenawSTX**
Adding security policy.

### CODE REVIEW

**Comment by wileyj** [comment]
Added some additional content here based on reviewing how other projects handle this doc.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+
+## Supported Versions
+Please see [Releases](https://github.com/stacks-network/stacks-blockchain/releases). We recommend using the [most recently released version](https://github.com/stacks-network/stacks-blockchain/releases/latest).
+
+
+## Reporting a vulnerability
+**Please do not file a public issue or PR mentioning the vulnerability.**
+
+We’ve partnered with [ImmuneFi](https://immunefi.com/) to reward honest researchers who find and responsibly disclose security vulnerabilities in our critical code. We offer bounties payable in the Stacks token ([STX](https://www.coingecko.com/en/coins/stacks)) for accepted, high-quality submissions.
```
Comment (pavitthrap) [inline SECURITY.md:11]: r/We/The Stacks Foundation

^ this substition applies for the entire PR.

Comment (wileyj) [reply SECURITY.md]: Language has been adjusted

**Review by kantai — APPROVED** [review]
LGTM

**Review by fpbgg — APPROVED** [review]

**Review by wileyj — APPROVED** [review]
shipit

**Pull request merged**